WebAPI/CurrencyConverter/DTO/CurrencyDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CurrencyConverter.DTO
+{
+    public class CurrencyDto
+    {
+        public string Base { get; set; }
+        public DateTime Date { get; set; }
```

Comment:
Can we use "DateOnly" type? As I've seen, there is only date without time in object which Danil showed us.
What would you suggest, do we need DateTime, or we can use DateOnly?
@Danil-Dovhopolyi